src/pages/spawn/spawnHost/SpawnHostTableActions.tsx

```diff
@@ -59,8 +59,8 @@ export const CopySSHCommandButton: React.FC<{
             size={Size.XSmall}
             data-cy="copy-ssh-button"
           >
-            <Label>Copy SSH command</Label>
-          </PaddedButton>
+            <Label>Copy SSH Command</Label>
```

Comment:
The Button is so large compared to the other ones. Can we check with design to see if this is ok.
![image](https://user-images.githubusercontent.com/4605522/133487314-21a50fac-292d-4018-a6c0-796bf19a3269.png)

Reply:
Agreed, maybe we could change the copy to something shorter. I'll add it to tomorrow's agenda!